### PR TITLE
Yamls for MIRS type sea-ice obs in cycling test

### PR DESCRIPTION
**Pull request by apchoiCMD**
#### This PR is to add the yaml key for cycling test

- Yaml keys for MIRS_NPP Ioda Converters
- Generate `gdas.t06z.icec_mirs.2021063006.nc4` for the cycling test

#### Result looks like below
GFSv17-3DVAR-C384mx025_gdasprepoceanobs_202106300600

### CODE REVIEW

**Review by JohnSteffen-NOAA — DISMISSED** [review]
Looks good.

**Review by guillaumevernieres — COMMENTED** [review]
Looks fine to me but was this tested @apchoiCMD ?

**Review by AndrewEichmann-NOAA — DISMISSED** [review]
looks good

File: parm/soca/obs/obs_list.yaml [hunk 1]
```diff
@@ -15,6 +15,7 @@ observers:
 # Ice concentration
 - !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+- !INC ${OBS_YAML_DIR}/icec_mirs.yaml
```
Comment (guillaumevernieres) [inline parm/soca/obs/obs_list.yaml]: could you comment that out @apchoiCMD , I don't think we want this to be part of the default obs list setup yet.

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
Sorry for the late review @apchoiCMD , just a minor comment related to the obs list.

File: parm/soca/obs/obs_list.yaml [hunk 2]
```diff
@@ -15,6 +15,7 @@ observers:
 # Ice concentration
 - !INC ${OBS_YAML_DIR}/icec_amsr2_north.yaml
 - !INC ${OBS_YAML_DIR}/icec_amsr2_south.yaml
+#- !INC ${OBS_YAML_DIR}/icec_mirs.yaml
```
Comment (guillaumevernieres) [inline parm/soca/obs/obs_list.yaml]: sorry for catching this just now @apchoiCMD , but the file names above do not fit our convention, see how sst is constructed: {variable}_{instrument}_{satellite}_{level}.yaml
We'll need to make these changes to the AMSR2 yaml as well but in a different PR.

**Review by guillaumevernieres — CHANGES_REQUESTED** [review]
The naming convention you use is not correct, see comment.

File: parm/soca/obs/obs_list.yaml (hunk 2 above)
Comment (apchoiCMD) [reply parm/soca/obs/obs_list.yaml]: Good point, I will modify accordingly-

Comment (guillaumevernieres) [reply parm/soca/obs/obs_list.yaml]: sorry, I'm just realizing that the markdown rendering didn't do what I expected.
We should follow the convention we have for the sst yaml's, for example:
`sst_avhrr_ma_l3u.yaml`
variable_instrument_satellite_level.yaml

Comment (apchoiCMD) [reply parm/soca/obs/obs_list.yaml]: made changes and could you please take a look?

**Review by AndrewEichmann-NOAA — DISMISSED** [review]
Looks right

**Review by guillaumevernieres — APPROVED** [review]
Looks good, thanks @apchoiCMD .

**Review by JohnSteffen-NOAA — APPROVED** [review]

**Pull request merged**